docs/package.json

```diff
@@ -11,7 +11,8 @@
     "clear": "docusaurus clear",
     "lint": "eslint . --ext js,jsx,md,mdx",
     "prettier:check": "prettier --check .",
-    "prettier:format": "prettier --write ."
+    "prettier:format": "prettier --write .",
+    "typecheck": "tsc"
```

Comment:
If you just intend to check types and not compile anything, I think you want to add `--noEmit` here. (I would be totally in favor of switching all the JS files to TS though.)

Reply:
All of that is in the `tsconfig.json` file.

Reply:
Oh, I missed that. If we did want to use TypeScript elsewhere I think we'd need to remove that from tsconfig.json and then use --noEmit here, but that would be something for later.

Reply:
Ok, good note, thanks.